jobma/permissions.py

```diff
@@ -0,0 +1,25 @@
+"""jobma permission classes"""
+import logging
+
+from django.conf import settings
+
+from rest_framework.permissions import BasePermission
+
+
+log = logging.getLogger(__name__)
+
+
+class JobmaWebhookPermission(BasePermission):
+    """Restrict access to jobma via access token"""
+
+    def has_permission(self, request, view):
+        header = request.headers.get("authorization", "")
+        if header.startswith("Token "):
```

Comment:
I think this should assert on the webhook access token so if the var isn't set, it's not possible for an empty bearer token to successfully authenticate:

```suggestion
        if settings.JOBMA_WEBHOOK_ACCESS_TOKEN and header.startswith("Token "):
```

Reply:
Oh sorry, I had this in a commit but I forgot to push it. What do you think of the new commit?